### PR TITLE
Hotfix/lst 1328 GitHub vulnerability scan

### PR DESCRIPTION
**Pull request by Luisella21**
Several changes due to security alert.
Important: the sql script must be executed before the migrations.

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -207,3 +195,32 @@
     ("scss", "SCSS"),
     ("yaml", "YAML"),
 )
+
+AUTHBROKER_ANONYMOUS_PATHS = ("check",)
+
+LOGGING = {
```
Comment (rossmiller) [inline config/settings/base.py:201]: We need to change or remove this logging config.

File: config/settings/env.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "ecs_formatter": {"()": ECSFormatter,},
+        "simple": {"format": "{asctime} {levelname} {message}", "style": "{",},
+    },
+    "handlers": {
+        "ecs": {
+            "class": "logging.StreamHandler",
+            "stream": sys.stdout,  # noqa F405
```
Comment (rossmiller) [inline config/settings/env.py:29]: Could you please remove lines 29 and 34?

**Review by rossmiller — CHANGES_REQUESTED** [review]
Looks very good! Some changes needed to logging (which is my fault for not explaining more).

File: config/settings/base.py (hunk 1 above)
Comment (Luisella21) [reply config/settings/base.py:201]: I removed it.

File: config/settings/env.py (hunk 2 above)
Comment (Luisella21) [reply config/settings/env.py:29]: Done.

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**